documentation/prettier/readme.md

```diff
@@ -0,0 +1,57 @@
+# Prettier Config Documentation
+
+> Documentation for the `@strv/prettier-config` module.
+
+## Usage
+
+### Installing
+
+Run:
+
+```bash
+# Yarn:
+yarn add --dev @strv/prettier-config
+
+# npm:
+npm install --save-dev @strv/prettier-config
+```
+
+And reference it in your `package.json` file:
+
+```js
+{
+  // ...
+  "prettier": "@strv/prettier-config"
+}
+```
+
+_Note: If you are using `prettier` < v1.17 check [this note](#prettier-117)._
+
+### <a id="extending"></a> Extending
```

Comment:
I think you do not need `<a>` tags to use relative links, at least on Github you can just link to that heading by using the heading's text, instead. Not sure how the conversion to a URL-safe string works but at least in this simple example you can just use `#Extending` and it should work™.